src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -118,9 +118,6 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
   struct VariablesToObserve {
     static constexpr Options::String help = "Subset of variables to observe";
     using type = std::vector<std::string>;
-    static type default_value() noexcept {
-      return {db::tag_name<Tensors>()...};
-    }
```

Comment:
should this be a suggested value, so one gets the information what other variables are available when some are omitted? This could save someone a rerun of a simulation if they forgot to add a variable to this list.